.meteor/packages

```diff
@@ -14,3 +14,4 @@ mantle:fixtures
 # CollectionFS security requires this to be global
 # https://github.com/CollectionFS/Meteor-CollectionFS/issues/711
 accounts-base
+useraccounts:core@1.12.2
```

Comment:
Is there some reason this needs to be in .meteor/packages? I'd probably put it in controllers, or some other package.